nengo_loihi/tests/test_neurons.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import nengo
+import pytest
+
+from nengo_loihi.neurons import get_loihi_rates
+
+
+@pytest.mark.parametrize("neuron_type", [
+    nengo.LIF(),
+    nengo.LIF(tau_ref=0.001, tau_rc=0.07, amplitude=0.34),
+    nengo.SpikingRectifiedLinear(),
+    nengo.SpikingRectifiedLinear(amplitude=0.23),
+])
+def test_loihi_rates(neuron_type, Simulator, plt, allclose):
+    n = 256
+    x = np.linspace(0, 1, n)
+
+    encoders = np.ones((n, 1))
+    max_rates = 400 * np.ones(n)
+    intercepts = 0 * np.ones(n)
+    gain, bias = neuron_type.gain_bias(max_rates, intercepts)
+    j = x * gain + bias
+
+    with nengo.Network() as model:
+        a = nengo.Ensemble(n, 1,
+                           neuron_type=neuron_type,
+                           encoders=encoders,
+                           gain=gain,
+                           bias=j)
+        ap = nengo.Probe(a.neurons)
+
+    dt = 0.001
+    with Simulator(model, dt=dt) as sim:
+        sim.run(1.0)
+
+    est_rates = sim.data[ap].mean(axis=0)
+    ref_rates = get_loihi_rates(neuron_type, x, gain, bias, dt=dt)
+
+    # plt.plot(x, ref_rates, "k", label="predicted")
```

Comment:
OK to remove these?

Reply:
I actually bring them back in #55. They're the better way to do it, because they work even if part of `x` is negative. The `semilog` just helps because so much of the interesting LIF stuff is crammed into the lower part of the curve. But really I think we should go back to just using `plot`.